motioneye/extra/linux_init

```diff
@@ -59,13 +59,19 @@ else
 fi
 
 # Pre-create config and data dirs and install configuration files
-[[ -d '/etc/motioneye' ]] || mkdir /etc/motioneye
-[[ -f '/etc/motioneye/motioneye.conf' ]] || cp extra/motioneye.conf.sample /etc/motioneye/motioneye.conf
+if [[ -f '/etc/motioneye/motioneye.conf' ]]
+then
+  # Update PID file directory if the default from old motionEye is still present: https://github.com/motioneye-project/motioneye/issues/2657
+  grep -q '^run_path /var/run$' /etc/motioneye/motioneye.conf && sed -i '/^run_path /var/run$/c\run_path /run/motioneye' /etc/motioneye/motioneye.conf
```

Comment:
When dry-testing this (running the command manually on a copy of the real one) locally on my Raspbian box I get:
```
sed: -e expression #1, char 31: expected newer version of sed
```
I haven't ever been able to make heads or tails of `sed` syntax, so I can't really tell what's wrong.

```
$ cat /etc/debian_version 
9.6
$ sed --version
sed (GNU sed) 4.4
```

Same happens on my Debian testing box.
```
$ cat /etc/debian_version 
12.0
$ sed --version
sed (GNU sed) 4.9
```

Reply:
Ah whoops, my fault, `/` as delimiter of course cannot work if used on the pattern as well.